### ISSUE TITLE
include some text for programming tools

### ISSUE THREAD

**Issue by cbm755** [hughsie/oars#27]
If the application being rated is a programming language or programming environment, then users could implement their own "Online Text-only Messaging" and similar things asked under "Social".

[Our interpretation](https://savannah.gnu.org/bugs/?56466) for GNU Octave is that the answer is `None` in this case.  
We think the question means "as provided by the application out-of-the-box."

Would it be useful to have some text in the webapp to help developers from bumping into this?

"If your game/application allows users to write their own scripts or code, then these questions would typically pertain to the out-of-the-box experience.  For example, if a game does *not* provide Online Audio and Video Messaging, but has an embedded Python console that could theoretically be used by end-users to implement video chat, the answer is probably `None`."

**Comment by hughsie** [comment]
> We think the question means "as provided by the application out-of-the-box."

I agree. I also think it's unlikely an embedded console would be used to implement a chat application, as you can take this to the extreme and argue that any app written in python or lua or shell could be rewritten to implement a chat function. I think the out-of-the box question is generic in all cases (i.e. you could use libreoffice-calc to draw porn, but it's unlikely) although I think maybe we should clarify things in the generic case.
